ansible/configs/ocp-clientvm-rhel8/software.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Step 00xxxxx software
+  hosts: localhost
+  gather_facts: False
+  become: false
+  tasks:
+    - debug:
+        msg: "Software tasks started"
+
+- name: Set up ClientVM
+  hosts: bastions
+  gather_facts: false
+  become: true
+  tasks:
+  - name: Set up Client VM for OCP 3
+    include_role:
+      name: "ocp-client-vm"
```

Comment:
If you don't specify a condition, it means it will always be included, so you can simply use `import_role` here.
That way it's included in the syntax-check tests too.

Reply:
Is there a benefit of include vs import ?

Reply:
changed.